lib/requester/core.js

```diff
@@ -439,7 +439,14 @@ module.exports = {
         // the SSL and TLS protocol versions to disabled during negotiation
         if (Array.isArray(protocolProfileBehavior.tlsDisabledProtocols)) {
             protocolProfileBehavior.tlsDisabledProtocols.forEach(function (protocol) {
-                options.secureOptions |= constants[SSL_OP_NO + protocol];
+                // since secure options doesn't support TLSv1.3 before Node 14
+                // @todo remove the if condition when we drop support for Node 12
+                if (protocol === 'TLSv1_3' && !constants[SSL_OP_NO + protocol]) {
+                    options.maxVersion = 'TLSv1.2';
```

Comment:
Since tests didn't cover this, have you ensure this works in Node v14?

Reply:
Yes, checked in Node v14. Works correctly without the `if` condition.